source/clear-linux/get-started/virtual-machine-install/vmware-esxi-install-cl.rst

```diff
@@ -0,0 +1,294 @@
+.. _vmware-esxi-install-cl:
+
+Install Clear Linux as a VMware\* ESXi guest OS
+###############################################
+
+`VMware ESXi`_ is a type 1 bare-metal hypervisor which runs directly on top of 
+server hardware.  With VMware ESXi, you can create, configure, manage, and run 
+|CLOSIA| virtual machines in the cloud.  
+
+This section shows you how to create a new :abbr:`VM (Virtual Machine)` and 
+install |CL| into it with VMware ESXi 6.5 Update 1.  Some of the advantages of 
+installing |CL| into a new VM is that it provides you flexibility in configuring, 
+for example: its size, the nummber of partitions, the initial of bundles, etc. 
+We will perform these steps: 
+
+#.  Download the latest |CL| installer ISO
+#.  Verify the integrity of the |CL| image
+#.  Uncompress the |CL| image
+#.  Upload the |CL| installer ISO to the VMware server
+#.  Create and configure a new VM
+#.  Install |CL| into the new VM
+#.  Reconfigure the VM's settings to boot the newly-installed |CL|
+#.  Power on the VM and boot |CL|
+
+If you would prefer to use a pre-configured |CL| VMware image instead, 
```

Comment:
I would consider changing this to "You can also use..." instead of "If you would prefer to use..."

Reply:
@rcaballeromx - have any opinion?